plugins/inputs/cassandra/cassandra_test.go

```diff
@@ -197,24 +197,25 @@ func TestHttpJsonJavaMultiType(t *testing.T) {
 }
 
 // Test that the proper values are ignored or collected
-func TestHttpJsonOn404(t *testing.T) {
+func TestHttp404(t *testing.T) {
```

Comment:
As a resolution to [the earlier comment on this test](https://github.com/influxdata/telegraf/pull/2372#discussion_r99757820), I'm changing the test to look for the error that is now passed via AddError(). Previous behavior of the plugin was that it was ignoring http-404 errors, and instead printing (not returning) an error when it failed to parse the response body. New behavior calls `AddError()` on both non-http-200 responses, as well as parse errors.

The cassandra plugin test was modified for the same reason.